std/src/main/scala/algebra/std/int.scala

```diff
@@ -8,6 +9,11 @@ package object int extends IntInstances
 
 trait IntInstances {
   implicit val intAlgebra = new IntAlgebra
+
+  val IntMinMaxLattice = new Lattice[Int] {
```

Comment:
No real reason, I just threw this together for testing purposes. I'll probably generalize it to anything that has an order, TBH.